rules/build_electricity.smk

```diff
@@ -226,21 +226,19 @@ rule determine_availability_matrix_MD_UA:
         country_shapes=resources("country_shapes.geojson"),
         offshore_shapes=resources("offshore_shapes.geojson"),
         regions=lambda w: (
-            resources("regions_onshore_base_s_{clusters}.geojson")
+            resources("regions_onshore.geojson")
```

Comment:
Not ideal, since previously the unclustered regions were called `regions_onshore.geojson` (i.e. same file name, different content). Perhaps `regions_onshore_clustered.geojson`?